reference-implementation/lib/parser.js

```diff
@@ -113,6 +117,10 @@ function normalizeSpecifierKey(specifierKey, baseURL) {
 
   const url = tryURLLikeSpecifierParse(specifierKey, baseURL);
   if (url !== null) {
+    if (url.protocol === BUILT_IN_MODULE_PROTOCOL && url.href.includes('/')) {
+      console.warn(`Invalid specifier key "${url.href}". Built-in module URLs must not contain "/".`);
+      return null;
+    }
```

Comment:
Nit: perhaps abstract into a `validateBuiltinURL` function to avoid repetition of the error message.

Reply:
Ahh, I see you've made the message specific to the case. NVM.